apps/src/javalab/Playground.js

```diff
@@ -106,13 +135,28 @@ export default class Playground {
       // can't remove items if game is over
       return;
     }
+    if (this.imageItemExists(itemData)) {
+      this.imageItemIds.splice(this.imageItemIds.indexOf(itemData.id), 1);
+      this.removePlaygroundItem(itemData.id);
+    }
+    // TODO: handle text deletion
```

Comment:
Just curious, how are you thinking we'll handle deleting text? Could we handle it in a similar way as images and just add it to the same lists/playground state or do we need to handle deleting it separately?

Reply:
I think we'll be able to do the same thing. My thought it we could have a PlaygroundText component that handles displaying text similar to PlaygroundImage